ingress/controllers/gce/controller/utils.go

```diff
@@ -70,6 +72,21 @@ func (ing ingAnnotations) staticIPName() string {
 	return val
 }
 
+func (ing ingAnnotations) ingressClass() string {
+	val, ok := ing[ingressClassKey]
+	if !ok {
+		return ""
+	}
+	return val
+}
+
+// isGCEIngress returns true if the given Ingress either doesn't specify the
+// ingress.class annotation, or it's set to "gce".
+func isGCEIngress(ing *extensions.Ingress) bool {
+	class := ingAnnotations(ing.ObjectMeta.Annotations).ingressClass()
```

Comment:
Check for nil here would be better than checking for nil after every type assertion.